llpc/unittests/context/testOptLevel.cpp

```diff
@@ -59,6 +57,9 @@ TEST(LlpcContextTests, MatchPipelineOptLevel) {
   for (auto optLevel :
        {CodeGenOptLevel::None, CodeGenOptLevel::Less, CodeGenOptLevel::Default, CodeGenOptLevel::Aggressive}) {
 #endif
+
+    Context *context = new Context(GfxIp);
```

Comment:
Not new code, but why isn't this a non-pointer local variable?

Reply:
Good point - fixed.

Reply:
Thanks!